sqlalchemy_history/builder.py

```diff
@@ -217,3 +220,17 @@ def create_association_proxies(self, version_classes):
                     attr,
                     property(fget=proxy.__get__, fset=proxy.__set__, fdel=proxy.__delete__),
                 )
+
+    def create_hybrid_property(self, version_classes):
+        """
+        Create Hybrid Property for Column as a property in Versioned Models from Original Model
+        """
+        for cls_ in version_classes:
+            hybrid_property_maps = get_hybrid_properties(cls_)
+            for attr, prop in hybrid_property_maps.items():
+                versioned_target_class = version_class(cls_)
+                setattr(
+                    versioned_target_class,
+                    attr,
```

Comment:
fset and fdel are most likely not required here also - as we the history information is readonly.